test/operators.jl

```diff
@@ -1,3 +1,16 @@
+import LightGraphs.edges, LightGraphs.nv, LightGraphs.add_edge!
+
+abstract type DummyAbstractGraph{T} <: AbstractGraph{T} end
+
+mutable struct DummyGraph{T} <: DummyAbstractGraph{T}
+    SG::SimpleGraph{T}
+end
```

Comment:
We define `DummyAbstractGraph` in two different locations. Wouldn't it be better to have a `testutils.jl` (or similar name) file, where we define such things?

Reply:
Yes the build is also failing due to redefinition of DummyGraph (should have used a different name) but creating testutils.jl is a better idea

Reply:
But that should be part of a separate PR I guess, @sbromberger views ?

Reply:
@abhinavmehndiratta you can just change the name here to `NewDummyGraph`, this would do it

Reply:
Okay I'll fix it for now :smile: